packages/mobile/src/send/SendConfirmation.tsx

```diff
@@ -357,8 +364,18 @@ export class SendConfirmation extends React.Component<Props, State> {
             feeLoading={asyncFee.loading}
             feeHasError={!!asyncFee.error}
             totalFee={fee}
+            currencyInfo={{
```

Comment:
Consider defining `currencyInfo` once up top then referencing that variable throughout these prop assignments